src/ferc_xbrl_extractor/datapackage.py

```diff
@@ -391,9 +391,7 @@ def __init__(self, schema: Schema, period_type: str):
         """Create FactTable and prepare for constructing dataframe."""
         self.schema = schema
         # Map column names to function to convert parsed values
-        self.columns = {
-            field.name: CONVERT_DTYPES[field.type_] for field in schema.fields
-        }
+        self.columns = {field.name: field.type_ for field in schema.fields}
```

Comment:
Change this to map column names to dtype labels like `"string"`, or `"integer"`. This allows us to use this dictionary to get both types to parse values, and pandas dtypes.